app/src/main/java/com/github/se/icebreakrr/ui/sections/Notifications.kt

```diff
@@ -45,7 +44,7 @@ fun NotificationScreen(navigationActions: NavigationActions, profileViewModel: P
         BottomNavigationMenu(
             onTabSelect = { route -> navigationActions.navigateTo(route) },
             tabList = LIST_TOP_LEVEL_DESTINATIONS,
-            selectedItem = Route.NOTIFICATIONS)
+            selectedItem = navigationActions.currentRoute())
```

Comment:
wrong fix: Thanks for solving my issue, but on friday's meeting the coaches told us it was better to have the current screen's icon highlighted in the navigation menu. You need to unfix this and apply the selection highlight for every screen's icon. you will also need to make the icon's background less bright

![image](https://github.com/user-attachments/assets/bf0d68e2-c773-4828-bdff-eb582ef2fec7) TOO WHITE!